### ISSUE TITLE
If DataDir doesnt exist, create it

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2359]
New puppeteer throws errors if the folder is not created before launching the browser

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.27.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.27.1...4.27.2) (2021-12-17)

- Release 4.27.2 [`c2b4e37`](https://github.com/open-wa/wa-automate-nodejs/commit/c2b4e37f38f680284a76f79dfd239be8a18658a3)
- 🐛 fix: if dataDir doesnt exit, create it #2359 [`b533cc2`](https://github.com/open-wa/wa-automate-nodejs/commit/b533cc219a6483fca9813deefc44b7ed4ca42a22)
- updated types-only package [`419ccfb`](https://github.com/open-wa/wa-automate-nodejs/commit/419ccfb5b288ed0e1772206c9bfbe2c55bc82bc3)
